Automattic/jetpack
# Search: Try rendering dashboard POC via class within helper package

Follow-up for #21339.

#### Changes proposed in this Pull Request:
This is a proof-of-concept PR.

* Adds a Dashboard PHP class to the Search package.
* Adds a Webpack build process for the Dashboard within the Search package.
* Adds a (duplicated) functioning Search Dashboard to wp-admin. Note that this Dashboard lacks the data layer integration necessary for enabling/disabling Search and Instant Search.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your local Jetpack installation.
* Build the Search package via `pnpm build` in `projects/packages/search/`.
* Build the Jetpack plugin via `pnpm build` in `projects/plugins/jetpack/`.
* Navigate to `/wp-admin`. Ensure that there's a "Search 2" submenu within the Jetpack admin menu.
* Ensure that the Search 2 submenu renders a(n) (incomplete version of) the Jetpack Search dashboard.
